_data/simple-icons.json

```
@@ -9621,6 +9621,11 @@
             "hex": "37BEFF",
             "source": "https://roundcube.net/"
         },
+        {
+            "title": "rsocket",
```

```suggestion
            "title": "RSocket",
```

Pay attention to spelling.